test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegatePerInstanceInheritedData.py

```diff
@@ -106,27 +113,26 @@ def testPerInstanceInheritedData(self):
 
         # These tests behave differently before USD version 21.05, so don't run
         # them for those earlier versions.
-        if Usd.GetVersion() < (0, 21, 5):
+        if usdVersion < (0, 21, 5):
             return
 
-        # There was a bugfix in USD version 21.08 that changes the results of these
-        # tests. Disable the tests while I work to update them.
-        if Usd.GetVersion() >= (0, 21, 8):
-            return
+        imageVersion = None
+        if usdVersion < (0, 21, 8):
+            imageVersion = 'pre-21_08'
```

Comment:
There are now two sets of images for the tests that follow this code. The main set in the regular folder is what the most up-to-date results are. The older images are in the pre-21_08 folder. In the future when we don't support any USD versions older than 21.08 we can remove those old images.